### ISSUE TITLE
[FEATURE REQUEST] Make Ansible code compliant with ansible-lint rules

### ISSUE THREAD

**Issue by to-bar** [epiphany-platform/epiphany#1177]
**Is your feature request related to a problem? Please describe.**
We've added `ansible-lint` to devcontainer and want to have code compliant with `ansible-lint` rules.

**Describe the solution you'd like**
No problems reported by `ansible-lint`.

Ansible-lint results as per 09/16/2021, probably want to re-run:
[ansiblelint_results.txt](https://github.com/epiphany-platform/epiphany/files/7176539/ansiblelint_results.txt)

---

**DoD checklist**

* [ ] Changelog updated (if affected version was released)
* [ ] COMPONENTS.md updated / doesn't need to be updated
* [x] Automated tests passed (QA pipelines)
  * [x] apply
  * [x] upgrade
* [x] Case covered by automated test (if possible)
* [x] Idempotency tested
* [ ] Documentation updated / doesn't need to be updated
* [ ] All conversations in PR resolved
* [ ] Backport tasks created / doesn't need to be backported

**Comment by rafzei** [comment]
I think its no longer valid - can you confirm @to-bar ?

**Comment by to-bar** [comment]
@rafzei It's still valid, here is an example: https://github.com/epiphany-platform/epiphany/blob/3e68564a9d765aa824382c03b8c3f8c65a49e1c5/core/src/epicli/data/common/ansible/playbooks/roles/image_registry/tasks/main.yml#L2

I think we could introduce the use of Ansible Lint (https://docs.ansible.com/ansible-lint/) and agree on the rules we want to follow.

**Comment by rafzei** [comment]
Default ansible-lint rule for checking if name property exist:
```
502: All tasks should be named
  All tasks should have a distinct name for readability and for ``--start-at-task`` to work
```

**Comment by mkyc** [comment]
Should we rename it to something about Ansible lint?

**Comment by to-bar** [comment]
> 
> 
> Should we rename it to something about Ansible lint?

IMHO the name is fine but it could be easier to do this task after #1176.

**Comment by mkyc** [comment]
It doesn't look like task for me but rather section in "code quality" document and part of DoD Check.

**Comment by to-bar** [comment]
> 
> 
> It doesn't look like task for me but rather section in "code quality" document and part of DoD Check.

It is a task since some existing Ansible tasks are missing the name property. For improving code quality there is separate task (#1176 ). Introducing Ansible Lint we can likely ensure DoD check in this area automatically (with pipelines).

**Comment by asciible** [comment]
Waiting for completion of https://app.zenhub.com/workspaces/epiphany-platform-5ef9d3c749afc20019782391/issues/epiphany-platform/epiphany/2847

**Comment by rafzei** [comment]
@asciible hi, are you working on this issue? I see it's in todo and you are assigned.

**Comment by asciible** [comment]
I haven't started working on it. I was planning to start next week. Though you can take if you want.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
